Makefile

```diff
@@ -125,6 +125,10 @@ tmp/$(HOST)-$(PORT).key tmp/$(HOST)-$(PORT).crt: ## Self-signed cert for local H
 run: ## Runs the development server
 	foreman start -p $(PORT)
 
+urn:
+	@echo "⚱️"
```

Comment:
among the billions of typos I make every day, I do not make this typo, I would not necessarily add this to the Makefile.

HOWEVER, now that this is here, I say we keep it just for the emoji urn ⚱️

Reply:
Emojis turn sad typo 😩  into happy typo 😄